.travis.yml

```diff
@@ -1,24 +1,36 @@
 sudo: false
 language: node_js
-node_js:
-- '8'
-- '10'
-- '11'
 cache:
   directories:
   - node_modules
-before_script:
-- yarn run lint
-script:
-- yarn run coverage
-- if [ -n "${FRONTEND-}" ]; then yarn run test-frontend ; fi
-after_script:
-- yarn run codecov
-matrix:
+stages:
+  - lint
+  - test:frontend:hooks
```

Comment:
what about unit tests?  We have to run unit tests on 8, 10 and 12 (remove 11) and also frontend:hooks can run only on any single version, It is not really required since it does not change the functionality.

Reply:
> what about unit tests? We have to run unit tests on 8, 10 and 12 (remove 11) 
Added

> also frontend:hooks can run only on any single version, It is not really required since it does not change the functionality.

I didn't get this one

Reply:
`npm run test` -> unit tests, We don't need to run `npm run test:frontend:hooks` on all versions.

Reply:
Which one do you recommend? I would keep it to the latest, in this case 12